draft-ietf-quic-transport.md

```diff
@@ -926,6 +926,23 @@ them with other connection IDs for the same connection.  As a trivial example,
 this means the same connection ID MUST NOT be issued more than once on the same
 connection.
 
+Packets with long headers include at least two connection IDs, which are used to
+establish the connection IDs that are used for the connection when established,
+see {{negotiating-connection-ids}} for details.
```

Comment:
Please consider rewording. The layers of establishment are confusing. Also, are the contained ID's used to establish themselves, or something else? If there are at least two, how many might that be in total, and why?

Reply:
There can be three in a Retry.

Reply:
How about "Packets with long headers include two connection IDs in the header, which are used.."  If you want to clarify that Retry adds a third used for path verification, that would be better than saying "at least 2"

Reply:
"used" is used twice in the sentence, which doesn't reads well. Also agree with @ianswett that it is worth spelling out the Retry case instead of alluding to it.

Reply:
Not at this point in the document; this is too soon for that.  Tweaked.